examples/neuron_multi_node/run_llama_3_70b_2_neuron_nodes.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -x #echo on
+
+# Use environment variables with defaults
+
+if [ -z "$K8S_MASTER_ADDR" ]; then
+	MASTER_ADDR=(`scontrol show hostnames $SLURM_JOB_NODELIST`)
+else
+	MASTER_ADDR=$K8S_MASTER_ADDR
+fi
+NEURON_RT_ROOT_COMM_ID=$MASTER_ADDR:8990
+NEURON_RANK_ID=${K8S_NEURON_RANK_ID:-$SLURM_NODEID}
+NEURON_LOCAL_TP=${K8S_NEURON_LOCAL_TP:-32}
+VLLM_HOST_IP=$MASTER_ADDR
+VLLM_PORT=8989
+
+export NEURON_RT_ROOT_COMM_ID
+export NEURON_RANK_ID
+export NEURON_LOCAL_TP
+export VLLM_HOST_IP
+export VLLM_PORT
+
+echo $NEURON_RT_ROOT_COMM_ID
+echo $NEURON_RANK_ID
+echo $NEURON_LOCAL_TP
+echo $VLLM_HOST_IP
+echo $VLLM_PORT
+
+# Install RT (commented out for safety)
+echo "running script"
+if [ -n "$SLURM_NODEID" ]; then
+	sudo dpkg -i ./neuron_dependencies/*.deb
+fi
```

Comment:
i guess we don't need these local setup in the upstream.

Reply:
```
export NEURON_RT_ROOT_COMM_ID
export NEURON_RANK_ID
export NEURON_LOCAL_TP
export VLLM_HOST_IP
export VLLM_PORT
```

These are required for multi node inference using transformers-neuronx library. I can get rid of the RT install part.